src/Void.Core/MessageLog.fs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Void.Core
```

Comment:
Should this be in `Void.Core`? This feels like a feature of the messaging system itself, which is implemented in `Void`, not part of the core editor. Ultimately, perhaps the top-level `Void` assembly is not the best place for the messaging stuff; there may be a `Void.Messaging` assembly or something. But in the mean time, `Void.Core` is growing towards just being core editor logic. This seems like more of an infrastructure thing.

Other than that, though, this file looks good. I dig the simplicity.